week6/assignment2_1/includes/ex.hpp

```diff
@@ -0,0 +1,31 @@
+#ifndef EX_HPP
+#define EX_HPP
+#include <iostream>
+#include <limits>
+
+class Positive_integers {
```

Comment:
I'd change the name to `Positive_integer`, since when you create an instance of a class, that is what you have.